### DISCUSSION TITLE
Is there a way to disable the order place email notifications?

### DISCUSSION THREAD

**Discussion by arbidridi** [pretix/pretix#2187] [Q&A]
Even making the email content empty in settings doesn't help. 

I've looked in event, org and global settings. 

Self-hosted Pretix v3.8.

**Comment by raphaelm** [comment]
Are you refering to the emails sent to the customer upon order creation? It is intentionally impossible to turn them off.

**Reply by arbidridi** [reply]
Yes, that's what I'm referring to. Just curious why customers must get two emails on an order with paid product. One before they pay and one after.

For now, I will check if I can block those outgoing emails at the Google SMTP level. Doesn't seem possible but I'll keep looking for workarounds.

**Reply by raphaelm** [reply]
If the payment fails or takes longer to confirm than expected (which can happen with almost any payment method), the first email allows to retry or view the status. If the first email wasn't sent, users might get lost in that situation.

**Reply by mariomnts** [reply]
We are looking to integrate our own emails using your webhooks because we need to add some custom content and design and it'd be great to be able to suppress all or only certain of these automatic communications.

**Reply by giacomobartoloni** [reply]
@arbidridi did you manage to disable the "order placed" email?

**Reply by sweenu** [reply]
I was wondering the same thing.

Ideally, for a payment provider that redirects you to the payment page straight away, I would love to be able to hold off the sending of the order confirmation email for a certain amount of time. If the user pays straight away, then send one email confirming both the order and payment. If however, it takes longer than the amount of time set, then the order confirmation email would be sent by itself.

@raphaelm I don't know if what I'm proposing is an acceptable solution. If you think it could be, and you could be interested, then I'd gladly work on implementing it.

**Reply by raphaelm** [reply]
I don't think this is easy to implement robustly and correctly. Also, it would require more additional email layouts to configure, to make sure the texts make sense. Therefore I think we wouldn't merge it at this time.

**Reply by mariomnts** [reply]
> I don't think this is easy to implement robustly and correctly. Also, it would require more additional email layouts to configure, to make sure the texts make sense. Therefore I think we wouldn't merge it at this time.

Maybe it's easier to give the option not to send some emails instead of that extra work of email layouts
